app/controllers/build_commands_controller.rb

```diff
@@ -10,14 +10,10 @@ def show
 
   def update
     command = params[:command][:command]
-    if command.blank?
```

Comment:
do a `if blank && unused` check instead ... would love to not leave weird commands hanging around